app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title><%= t('title_tag') %></title>
```

Comment:
How about just using `<%= t("#{controller_name}.#{action_name}.title_tag") %>` here? This can be DRYd up perhaps with a simple helper, but it'll do the job nicely. The translations will have to be changed to include keys for the controller and action names, like `en.landing_page.show.title_tag`.